wof/examples/flask/odm2/timeseries/runserver_odm2_timeseries.py

```diff
@@ -2,9 +2,6 @@
 
 import wof.flask
 
-curr_folder = os.path.abspath(os.path.dirname(__file__))
-sys.path.insert(0, '%s/../../../' % curr_folder)
```

Comment:
@lsetiawan Actually, if you can review this right now, that will be SUPER HELPFUL!! I've finished reviewing this PR from @ocefpaf, and your comments on this one change is the only thing that's left before I can approve the PR -- which I've been holding up for 3 weeks :(. THANKS!